src/Themes/Custom/assets/public/images/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/android-chrome-192x192.png",
```

Comment:
Mag die voor Android geen transparantie bevatten?

Reply:
Blijkbaar wordt transparantie niet getoond als je op android "add to homepage" gebruikt. Daar is dat icoon voor dus ik vermoed dat het daarom is (overgenomen van oude sumo theme)